coding/python_style_guide.rst

```diff
@@ -469,22 +469,22 @@ This comment should appear after the ``def`` line.
 We follow `PEP 8ʼs naming conventions <https://www.python.org/dev/peps/pep-0008/#naming-conventions>`_, with exceptions listed here.
 The naming conventions for LSST Python and C++ source have been defined to be as similar as the respective languages allow.
 
-In general:
+All LSST Python source code is consistent with PEP-8 naming as follows:
 
 - class names are ``CamelCase`` with leading uppercase,
```

Comment:
It's my fault, but since you're here, you could convert each of these list items into  complete sentences (start with a capital, end with a full stop). Thanks!